internal/install/types/types.go

```diff
@@ -104,86 +104,90 @@ var OpenInstallationTargetTypeTypes = struct {
 // OpenInstallationAttributes - Custom event data attributes
 type OpenInstallationAttributes struct {
 	// Built-in parsing rulesets
-	Logtype string `json:"logtype"`
+	Logtype string `json:"logtype,omitempty"`
 }
 
 // OpenInstallationLogMatch - Matches partial list of the Log forwarding parameters
 type OpenInstallationLogMatch struct {
 	// List of custom attributes, as key-value pairs, that can be used to send additional data with the logs which you can then query.
-	Attributes OpenInstallationAttributes `json:"attributes"`
+	Attributes OpenInstallationAttributes `json:"attributes,omitempty"`
 	// Path to the log file or files.
-	File string `json:"file"`
+	File string `json:"file,omitempty"`
 	// Name of the log or logs.
 	Name string `json:"name"`
 	// Regular expression for filtering records.
-	Pattern string `json:"pattern"`
+	Pattern string `json:"pattern,omitempty"`
 	// Service name (Linux Only).
-	Systemd string `json:"systemd"`
+	Systemd string `json:"systemd,omitempty"`
 }
 
 // OpenInstallationPreInstallConfiguration - Optional pre-install configuration items
 type OpenInstallationPreInstallConfiguration struct {
 	// Message/Docs notice displayed to user prior to running recipe
-	Prompt string `json:"prompt"`
+	Prompt string `json:"prompt,omitempty"`
 }
 
 // OpenInstallationRecipe - Installation instructions and definition of an instrumentation integration
 type OpenInstallationRecipe struct {
 	// Description of the recipe
 	Description string `json:"description"`
+	// Friendly name of the integration
+	DisplayName string `json:"displayName,omitempty"`
 	// The full contents of the recipe file (yaml)
 	File string `json:"file"`
 	// The ID
-	ID string `json:"id"`
+	ID string `json:"id,omitempty"`
 	// List of variables to prompt for input from the user
 	InputVars []OpenInstallationRecipeInputVariable `json:"inputVars"`
+	// Go-task's taskfile definiton (see https://taskfile.dev/#/usage)
```

Comment:
Not important but maybe to satisfy your linter, correct the spelling of `definiton` to `definition`

Reply:
Also, apologies, b/c this almost definitely came from you copy/pasting my spelling mistake 😂

Reply:
This code is all generated, so what is stored in the schema is what gets written here.  I'll correct it in place now.